charts/operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Can we move this to `keda-http-operator` please?

Reply:
Sure! I'll change it!

Reply:
Done